gapis/resolve/command_tree.go

```diff
@@ -319,6 +321,8 @@ func (r *CommandTreeResolvable) Resolve(ctx context.Context) (interface{}, error
 
 	for k, v := range snc.SubcommandGroups {
 		r := out.root.AddRoot([]uint64{uint64(k)})
+		// subcommands are added before nested SubCmdRoots.
+		sort.SliceStable(v, func(i, j int) bool { return len(v[i]) < len(v[j]) })
```

Comment:
I would say it is still just a quick-fix. The order should not matter for a more reasonable logic